tests/Browser/PublicViewTest.php

```diff
@@ -0,0 +1,72 @@
+    public function test_does_view_load(): void {
+        $this->browse(function (Browser $browser) {
+            $browser->visit('/')
+                    ->assertSee('Atlanta Virtual ARTCC');
+            $browser->visit('/pilots/scenery')
+                    ->assertSee('Scenery');
+            $browser->visit('/pilots/airports')
+                    ->assertSee('Airports');
+            $browser->visit('/pilots/request-staffing')
+                    ->assertSee('ZTL ARTCC Staffing Request');
+            $browser->visit('/pilots/guide/atl')
+                    ->assertSee('Atlanta Pilot Guide');
+            $browser->visit('/ramp-status/atl')
+                    ->assertSee("Atlanta Hartsfield Jackson Int'l Airport (ATL) Ramp/Gate Status");
+            $browser->visit('/ramp-status/clt')
+                    ->assertSee("Charlotte Douglas Int'l Airport (CLT) Ramp/Gate Status");
+            $browser->visit('/controllers/roster')
+                    ->assertSee('Roster');
+            $browser->visit('/controllers/staff')
+                    ->assertSee('Staff');
+            $browser->visit('/controllers/files')
+                    ->assertSee('ARTCC Files');
+            $browser->visit('/controllers/stats')
+                    ->assertSee('ARTCC Controller Statistics');
+            $browser->visit('/controllers/teamspeak')
+                    ->assertSee('ZTL ARTCC Teamspeak Information');
+            $browser->visit('/privacy')
+                    ->assertSee('ZTL ARTCC Privacy Policy & Terms and Conditions');
+            $browser->visit('/visit')
+                    ->assertSee('Visit ZTL ARTCC');
+            $browser->visit('/feedback/new')
+                    ->assertSee('Leave New Feedback');
+            $browser->visit('/trainer_feeback/new')
+                    ->assertSee('Leave New Training Team Feedback');
+            $browser->visit('/login')
+                    ->assertSee('Atlanta Virtual ARTCC wants to access your VATSIM account.');
+        });
+    }
+
+    public function test_external_links(): void {
+        $this->browse(function (Browser $browser) {
+            $browser->visit('https://www.aviationapi.com/charts')
+            ->assertDontSee('404');
+            $browser->visit('https://www.aviationweather.gov/')
+                    ->assertDontSee('404');
+            $browser->visit('http://www.flightaware.com/statistics/ifr-route/')
+                    ->assertDontSee('404');
+            $browser->visit('https://ids.ztlartcc.org/')
+                    ->assertDontSee('404');
+            $browser->visit('https://www.teamspeak.com/?tour=yes')
+                    ->assertDontSee('404');
+            $browser->visit('https://www.thepilotclub.org/')
+                    ->assertDontSee('404');
+            $browser->visit('http://www.vatsim.net/')
+                    ->assertDontSee('404');
+            $browser->visit('http://www.vatusa.net/')
+                    ->assertDontSee('404');
```

Comment:
I would argue for and against this...

For this - it's a good check on our end to see if external dependencies are working properly

Against this - it's using our resources to test something that we should be able to assume is working. If all of these services are testing properly, they should be working and really we shouldn't be testing external dependencies.

I'm not sure if I like this or not. If we decide to keep these and our tests get to a point where they take a long time, this should be the first to go.

Reply:
It's quick and gives us feedback regarding broken links. I like it, which is why I added it.

Reply:
I'm with Ian on this one. This could be a separate routine, maybe, but this makes our tests non-deterministic. Good tests should not rely on anything except our own website, except in a few cases (e.g. auth) where it can be justified, and the same test should always produce the same result given no change in the test code or the code it's testing - as that ensures that there are no false results.
Having stuff like this will likely result in causing us more harm than good, as say some internet weather causes one of these to break, we'd then have to rerun all of the checks on PRs that should have passed

Reply:
I think things like this are outside of the scope of what I originally intended when I wrote this issue. We can't *entirely* replace QA with testing - automated testing should be used for the stuff that can be easily automated (e.g. like 95% of the dashboard) and we should reserve the stuff that has external dependencies (I'm inclined to say the auth routine is part of this group) or can't easily be automated for manual QA.

Reply:
Tests have been broken out per our conversation and I eliminated testing external links